tests/test-utils.js

```diff
@@ -0,0 +1,21 @@
+class TestUtils {
```

Comment:
Do we really need to create a class with static functions for that?

We could just `export { waitExits, sleep }`

Reply:
I did created and exported a function, @jerome-benoit ( rightly IMO) pointed out to me that a class is easier to export/import.
I think we are good as is

Reply:
Ah I see... https://github.com/pioardi/poolifier/pull/172/commits/f47db8f6263a54ff3cbfe978de26e6c894e05ba7#diff-1f966d500f9582fd4290dcddfcfe799b8d4d6ca2f03015131aef6fb5fb581020

Ah maybe that's easier for JavaScript cause it's not `mjs`. I'm totally not used to JS anymore :smile: 
yeah okay, it's just tests :shrug: